the-collab-lab/tcl-33-smart-shopping-list
# Added green background to fieldset on homepage and updated tooltip position on navbar on the ViewList page

## Description



This code created a welcoming design for users on the homepage and also emphasizes the visual distinction between the create-list and join-list functionality of the app. Additionally, we updated the tooltip position in the navbar on the ViewList page.

## Related Issue

Closes #13 

## Acceptance Criteria

- Your team will self-organize around a lightweight design process to come up with a polished look and feel for your app.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before
![Screenshot (443)](https://user-images.githubusercontent.com/50176611/143923519-8356e53a-b767-45ae-b1af-a04a71983c3a.png)

### After
![Screenshot (445)](https://user-images.githubusercontent.com/50176611/143923723-49d138d6-1fc6-4c25-997c-fe6a2f4d5fba.png)

![homepage](https://user-images.githubusercontent.com/50176611/143924572-d3b5824b-eb9f-48f7-bbe0-247dee585125.gif)

## Testing Steps / QA Criteria
- Fork and clone https://github.com/the-collab-lab/tcl-33-smart-shopping-list
- In your terminal, enter "npm install" to install packages (if this is your first time forking/cloning)
- In your terminal, enter "npm start" to open the app locally in your browser
- If this is not your first time using this app, be sure to remove existing tokens from your local storage so that you can view the homepage without being redirected to the view list page
- You should see the pretty green band that separates the create new list and join list functionality!
